docs/sphinx/conf.py

```diff
@@ -140,6 +140,7 @@
     "databricks_api",
     "databricks_cli",
     "datadog",
+    "dlt",
```

Comment:
Alternatively, would adding it to `paths` resolve it? Similar to sling?
```
"../../python_modules/libraries/dagster-embedded-elt/sling",
```

Reply:
thats right. let me see if there's any weird transitive error once i add it there. if not, will update the pr.

Reply:
hmm still getting the error after adding:
```
"../../python_modules/libraries/dagster-embedded-elt/dlt",
```
looks like we are not installing the transitive dependency of those libs (e.g. even though `dagster-embedded-elt` depends on dlt, we don't really install dlt into the `sys.path`). so i believe adding `dlt` to the mock is still necessary to bypass that import error.

Reply:
but i added the /dlt path for consistency.